INSTALL.rst

```diff
@@ -90,8 +153,39 @@ Once the container is running, you should see a message like this:
 
 Locate the *last* URL in your terminal (the one that includes
 ``127.0.0.1``), and navigate to that URL in a web browser to access the
-Jupyter notebook interface.
+Jupyter Notebook interface.
 
 The home directory includes a subdirectory named ``persistent-volume``.
 All work you’d like to save should be placed in this directory, as it is
 the only one that will be saved across different container runs.
+
+
+Running some Examples
+^^^^^^^^^^^^^^^^^^^^^
+From inside the ``circuit_knitting_toolbox`` repository, navigate to the tutorials, open
+a `Jupyter Notebook <https://jupyter.org/install>`__ instance, and start experimenting with CKT!
+
+.. code::
+    
+    cd docs/<circuit_cutting | entanglement_forging>/tutorials
+    jupyter notebook
```

Comment:
Why not start `jupyter` from the base of the repository -- or at most, one level deep, in `docs`?

Reply:
Couldn't really settle on what was best. This section is kind of clunky anyway, but I think we need it to cap everything off.

What snippet would you suggest here?

Reply:
Maybe just

```
cd docs/
jupyter notebook
```

and then navigate thru the web interface to circuit_cutting or entanglement_forging, then tutorials